README.md

```diff
@@ -17,11 +17,11 @@ This is a react web app where users can look up and see details about countries
 
 ## Live Demo 
 
-[COMING SOON!]()
+[Country App](https://stupendous-klepon-6b06f0.netlify.app)
 
 ## Demo Video
 
-[Demo Video]()
+[Demo Video](https://www.loom.com/share/e73e6a5df5b2463db165293bcf33346f)
 
```

Comment:
- [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).